crates/rolldown/tests/rolldown/code_splitting/format-cjs-with-module-cjs/artifacts.snap

```diff
@@ -0,0 +1,57 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# Assets
+
+## main1.js
+
+```js
+"use strict";
+const require_share$1 = require('./share.js');
+const node_assert = require_share$1.__toESM(require("node:assert"));
+
+//#region main1.js
+var import_share = require_share$1.__toESM(require_share$1.require_share());
+node_assert.default.equal((0, import_share.share)(), 1);
+
+//#endregion
+```
+## main2.js
+
+```js
+"use strict";
+const require_share$1 = require('./share.js');
+const node_assert = require_share$1.__toESM(require("node:assert"));
+
+//#region main2.js
+var import_share = require_share$1.__toESM(require_share$1.require_share());
+node_assert.default.equal((0, import_share.share)(), 1);
+
+//#endregion
+```
+## share.js
+
+```js
+
+
+//#region share.js
+var require_share = __commonJS({ "share.js"(exports) {
+	exports.share = function share() {
+		return 1;
+	};
+} });
+
+//#endregion
+Object.defineProperty(exports, '__toESM', {
+  enumerable: true,
+  get: function () {
+    return __toESM;
+  }
+});Object.defineProperty(exports, 'require_share', {
```

Comment:
Not related to this PR: This could be improved.

Reply:
Yeah. Let's fixed it later.